Xamarin.Forms.Platform.iOS/Renderers/SwipeViewRenderer.cs

```diff
@@ -1172,6 +1173,14 @@ void ExecuteSwipeItem(ISwipeItem swipeItem)
 			swipeItem.OnInvoked();
 		}
 
+		bool ShouldVerifySwipeViewFlagEnabled()
+		{
+			if (UIApplication.SharedApplication?.Delegate?.GetType()?.FullName == "XamarinFormsPreviewer.iOS.AppDelegate")
```

Comment:
It's a good idea. Also, by default, using the Designer, we should ignore the validations of any Experimental flag, right?. If rendering using SwipeView was failing because of the Experimental flag, the same should happen in other cases like using IndicatorView, etc.

Reply:
I have added the extension method.